Apps/Immunization/src/appsettings.hgdev.json

```diff
@@ -16,5 +16,9 @@
     },
     "ClientAuthentication": {
         "TokenUri": "https://dev.oidc.gov.bc.ca/auth/realms/ff09qn3f/protocol/openid-connect/token"
+    },
+    "CDOGS": {
+        "DynamicServiceLookup": "false",
```

Comment:
Why are we not using the dynamic lookup?

Reply:
not sure just followed whats in the admin